Makefile

```diff
@@ -83,6 +83,7 @@ prereq: $(target/stamp-prereq) tmp/.prereq_packages
 		echo '       Please provide a "$(INCLUDE_DIR)/site/$(ARCH)" file and restart the build.'; \
 		exit 1; \
 	fi
+	$(_SINGLE)$(SUBMAKE) -r diffconfig
```

Comment:
Wrong place, I think. Prereq is run separately and also much too early. (E.g. first time already during feed update)

It should be in "prepare", not "prereq". See Stintel's example in https://github.com/lede-project/source/pull/673#issuecomment-270682184